scss/partials/_landingPages.scss

```diff
@@ -203,62 +203,6 @@ table {
   white-space: nowrap;
 }
 
-/* custom checkboxes (prototyped) */
```

Comment:
none of these are used anywhere

Reply:
Huh. That's funny. Maybe something older from kbase-ui?